packages/framework/src/Model/Mail/MailTemplateConfiguration.php

```diff
@@ -74,11 +74,14 @@ public function addMailTemplateVariables(
         $this->mailTemplateVariables[$mailTemplateSlug] = $mailTemplateVariables;
     }
 
-    protected function registerOrderStatusMailTemplates(): void
+    /**
+     * @return \Shopsys\FrameworkBundle\Model\Mail\MailTemplateVariables
+     */
+    protected function addOrderStatusMailTemplateVariables(): MailTemplateVariables
```

Comment:
I prefer a different name than `add*` for the methods responsible for creating the email variables. 

`get*MailTemplateVariables()` or `create*MailTemplateVariables()`

add seems like it adds variables somewhere, but in reality the mailtemplatevariables object is created and configured